src/utils/image-utils.js

```diff
@@ -0,0 +1,26 @@
+import ReactDOM from 'react-dom';
+
+export const preloadImage = async url => {
+  if (!url) {
+    return;
+  }
+
+  const element = window.document.createElement('img');
+  element.style.display = 'none';
+  element.src = url;
+
+  await new Promise(resolve => {
+    element.onloadeddata = () => {
+      ReactDOM.unmountComponentAtNode(element);
```

Comment:
I wonder why this is necessary? There's no React component involved here, just a DOM node (outside the React tree), so `.remove()` should be enough :thinking: 

(Also: this function will be renamed in the next React version)

Reply:
Edit: I realized that the element is never put into the document tree, so also `.remove()` is not necessary.